package.json

```diff
@@ -12,8 +12,9 @@
     "cover": "nyc --reporter=lcov --reporter=text npm t",
     "exec": "node dist/main.js",
     "lint": "npx eslint --fix src && npx eslint --fix test",
-    "test": "rimraf ./test/out && npm run build && npm run build:tests && mocha",
-    "rebuild:all": "npm run clean && npm run build:all"
+    "test": "rimraf ./out && npm run build && npm run build:tests && mocha ./out/ts/*.test.js",
+    "rebuild:all": "npm run clean && npm run build:all",
+    "e2e": "rimraf ./out && npm run build && npm run build:tests && mocha ./out/e2e-test/*.test.js"
```

Comment:
I believe we might be missing a full cleanup of the S3 storage. If yes, we could leave it to be done in in a another PR to allow us to close this one now.